src/style.toit

```diff
@@ -279,35 +279,40 @@ class RoundedCornerBorder extends InvisibleBorder:
     bottom := y2 + h2 - radius_
     if transparency-map is one-byte.OneByteCanvas_:
       palette := opacity == 0xff ? #[] : shadow-palette_
-      draw-corners_ x2 y2 right bottom radius_: | x y orientation |
-        transparency-map.pixmap x y
-            --pixels = opacities_.byte-opacity
-            --palette = palette
-            --source-width = radius_
-            --orientation = orientation
+      draw-corners_ x2 y2 right bottom radius_: | i j x y orientation |
+        byte-opacity := opacities_.byte-opacities_[(i << 16) + j]
```

Comment:
I would prefer not to reach into the `opacities` class.
Add two methods: `get-bytes-block` and `get-bits-block` (with the correct term for "block") that return the corresponding entry for `i` and `j`.